README.md

```diff
@@ -744,6 +744,27 @@ await database.UpsertAsync(people);
 
 ```
 
+
+#### Execute with temp table
+
+Kros.KORM offers special execute commands for SQL databases, that inserts provided simple data into temp table and
+then executes some specified action using those temporary data.
+You can find these extension methods in `IDatabaseExtensions` class.
+
+```CSharp
+database.ExecuteWithTempTable<TValue>(IEnumerable<TValue> values, action);
```

Comment:
Pridaj tu prosím nejaký príklad použitia, nech ten čo to ide použiť tuší ako to má použiť. Ja by som nevedel.